Website.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+import random
+
+
+app = Flask(__name__)
+random.seed(0)
```

Comment:
Typically, you want to seed it with some dynamic value, like the current time in milliseconds. Otherwise, every time you restart the site, the sequence of random numbers will be the same.